### ISSUE TITLE
Release with lanterna-native-integration

### ISSUE THREAD

**Issue by nicoulaj** [mabe02/lanterna#266]
Hi,

Do you plan on releasing a version to Maven Central including the `lanterna-native-integration` artifact anytime soon ?

I can see it is not configured as a submodule in the project, I can help with that if you need.

**Comment by mabe02** [comment]
Yes, at some point. It's still kind of experimental, I think it's doesn't work 100% on windows yet.

**Comment by nicoulaj** [comment]
I would like to help with this, if possible.
Do you plan to merge the `lanterna-native-integration` classes in the main jar, or turn the project into a multi-module one ?

**Comment by avl42** [comment]
I'm happy that there *is* some support in the making (even if it still needs some windows users to care, test and send feedback or patches).

But I'd prefer, if Lanterna itself remained "pure Java", only with an optional extra module to support those targets that aren't supportable without JNA.

I've heard the licensing terms of Oracle's JVM get pretty restrictive when native code gets involved.

**Comment by mabe02** [comment]
Yeah, I'm not going to include it in the main project. There are some hooks in the main code to detect if it's there, but that's about it. Might try to push it to Sonatype (under another artifact id) at some point if I can get it working again though.

**Comment by nicoulaj** [comment]
OK, I have created a branch that turns the project into a multi-module one without changing the main jar artifactId: https://github.com/nicoulaj/lanterna/tree/maven-multi-module

**Comment by avl42** [comment]
If there isn't a way to make it "multi-moduled" without moving every single file one level down, then I'd rather have an entirely separate "project" for the native parts. (That's just my opinion.)

**Comment by nicoulaj** [comment]
We can force the source directory to be on top level like it was, but it's really not good practice... Better make it two separate projects with two separate repositories in this case.

**Comment by mabe02** [comment]
Yeah, we've talked about this before, it should really be a separate repository. I should split it up.

**Comment by miho** [comment]
Any news on this? I would like to use lanterna for projects that run on Linux and Windows without using Swing.
